beater/handlers.go

```diff
@@ -341,7 +344,7 @@ func corsHandler(allowedOrigins []string, h http.Handler) http.Handler {
 			h.ServeHTTP(w, r)
 
 		} else {
-			sendStatus(w, r, forbiddenResponse)
+			sendStatus(w, r, forbiddenResponse(errors.New(origin+" is not allowed")))
```

Comment:
I would change this to something like `"origin: '" + origin + "' is not allowed"` in case the origin header is empty.